ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml

```diff
@@ -22,6 +26,9 @@ images:
     coreos_cosa-buildroot_latest:
       as:
       - registry.ci.openshift.org/coreos/cosa-buildroot:latest
```

Comment:
Should we remove this one?

Reply:
We can't because it'd break rehearsals here, so I'm multi-phase ratcheting:

- add the image here
- PR to rpm-ostree to use the new image
- drop the old image here